v21.1/connect-to-the-database-cockroachcloud.md

```diff
@@ -6,7 +6,7 @@ toc: true
 
 <div class="filters filters-big clearfix">
   <a href="connect-to-the-database.html"><button class="filter-button">Local</button></a>
-  <button class="filter-button current">{{ site.data.products.serverless }}</button>
+  <button class="filter-button current">{{ site.data.products.db }}</button>
```

Comment:
```suggestion
  <button class="filter-button current">{{ site.data.products.db }}</button></a>
```

Reply:
I think the `filter-button current` ones are all just button elements not urls

Reply:
> I think the `filter-button current` ones are all just button elements not urls

Yup, duh.